benchbuild/utils/actions.py

```diff
@@ -624,3 +624,28 @@ def __str__(self, indent: int = 0) -> str:
                 textwrap.indent("* Clean the directory: {0}".format(p),
                                 indent * " "))
         return "\n".join(lines)
+
+
+class ProjectEnvironment(Step):
+    NAME = 'ENV'
+    DESCRIPTION = 'Prepare the project environment.'
+
+    @notify_step_begin_end
+    def __call__(self):
+        project = self.obj
+        prj_vars = project.variant
+
+        for name, variant in prj_vars.items():
+            LOG.info(f"Fetching {str(name)} @ {variant.version}")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use lazy % formatting in logging functions](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)